client/components/marketing-survey/gsuite-cancel-purchase-dialog/gsuite-cancellation-features.jsx

```diff
@@ -17,28 +17,68 @@ class GSuiteCancellationFeatures extends Component {
 		this.props.recordTracksEvent( 'calypso_purchases_gsuite_remove_purchase_learn_more_click' );
 	};
 
+	getAccessMessage = () => {
+		const { googleSubscriptionStatus, purchase, translate } = this.props;
+		const { meta: domainName, productSlug } = purchase;
+
+		const googleMailService = getGoogleMailServiceFamily( productSlug );
+
+		if ( [ 'suspended', '' ].includes( googleSubscriptionStatus ) ) {
+			return translate(
+				'If you cancel your subscription for %(domainName)s now, {{strong}}you will lose access to all of ' +
+					'your %(googleMailService)s features immediately{{/strong}}, and you will ' +
+					'need to purchase a new subscription with Google if you wish to regain access to the features.',
+				{
+					args: {
+						domainName,
+						googleMailService,
+					},
+					comment:
+						'%(domainName) is the name of the domain, e.g. example.com; ' +
+						'%(googleMailService)s can be either "G Suite" or "Google Workspace" ',
+					components: {
+						strong: <strong />,
+					},
+				}
+			);
+		}
+
+		return translate(
+			'If you cancel your subscription for %(domainName)s now, {{strong}}you will lose access to all of ' +
+				'your %(googleMailService)s features in %(days)s days{{/strong}}. After that time, ' +
+				'you will need to purchase a new subscription with Google.',
+			{
+				args: {
+					domainName,
+					googleMailService,
+					days: '30',
+				},
+				comment:
+					'%(domainName) is the name of the domain, e.g. example.com; ' +
+					'%(googleMailService)s can be either "G Suite" or "Google Workspace" ' +
+					'and %(days)s is a number of days which will generally be 30, and will always be at least 10',
+				components: {
+					strong: <strong />,
+				},
+			}
+		);
+	};
+
 	render() {
 		const { purchase, translate } = this.props;
 		const { meta: domainName, productSlug } = purchase;
 
 		return (
 			<div className="gsuite-cancel-purchase-dialog__features">
 				<CardHeading tagName="h3" size={ 24 }>
-					{ translate( "Are you sure? Here's what you'll be missing:" ) }
+					{ translate( 'Are you sure?' ) }
 				</CardHeading>
 
-				<p>
-					{ translate(
-						'If you cancel and remove %(googleMailService)s from {{siteName/}} you will lose access to the following: ',
-						{
-							args: {
-								googleMailService: getGoogleMailServiceFamily( productSlug ),
-							},
-							comment: '%(googleMailService)s can be either "G Suite" or "Google Workspace"',
-							components: { siteName: <em>{ domainName }</em> },
```

Comment:
I am not really sure if should keep this HTML tag in the new `domainName` placeholder...